_posts/2021-11-06-explaining-signals-ui-change.md

```diff
@@ -0,0 +1,98 @@
+
+So block signals are faster? Yes and no.
+On a microscopic scale they can be, but they require a lot more care, and it's much easier to shoot yourself in the foot with block signals.
+We often see new players asking about how to make a station or interchange work, and then show a screenshot of a setup with very poorly placed signals, that would be much easier to build right with path signals.
+That's why we want to guide players, especially new players, to use path signals instead of block signals.
+
+Now let's look at some examples of common layouts with path signals.
+All of these examples use right-hand drive for the trains.
+
+![Terminus station with one line connecting]({% link /static/img/post_explaining-signals-ui-change/terminus-station-1.png %})
+
+This is a terminus station, with one line connecting to it, and a train depot.
+Because it's a terminus station, the points to select platform can be right next to the platform, no signal is needed between the platform and the points.
+There is a one-way path signal on the right-hand track going into the station area, which has a one-way path signal on it.
+This signal protects entry to the platforms and the depot.
+
+When a train reaches the signal, it looks for a free platform, and gets green light if there is a platform free.
+If there are no platforms free, the signal stays red and the train stops.
+As soon as a train has entered a platform and is free of the switching points, another train will be able to pass the points, either to enter or to leave the station.
+Trains only go to the depot if they are ordered to go there.
+
+This design is simple, compact, and good for moderate traffic.
+
+![Terminus station with two lines connecting]({% link /static/img/post_explaining-signals-ui-change/terminus-station-2.png %})
+
+This is a terminus station, with two lines connecting to it.
+It works very much like the one above, but it's larger and the switching area needs to take up more space.
+
+The signal setup works the same way:
+Trains entering or leaving the station look for a free path to go where they want.
+If there is a free path, the signal turns green, and the train can pass.
+If the path is blocked, such as if all platforms are blocked, or another train is already leaving the same direction, the train waits for the other train to clear the switching area.
+
+This design is also relatively simple and compact, but it does lose some capacity when several lines need to cross.
+
+![Through station with two platforms per direction]({% link /static/img/post_explaining-signals-ui-change/through-station.png %})
+
+This is a through-station on a line, with two platforms per direction on the line.
+There are one-way path signals at the exit of each platform, and one-way path signals before the switching point to select which platform to use.
```

Comment:
Maybe add something like this?

```suggestion
There are one-way path signals at the exit of each platform, and one-way path signals before the switching point to select which platform to use.
The signals at the platform exit are needed as otherwise the first train to enter a platform would also block the path out of the station up to the next signal.
The signal after the platform breaks this path and makes sure that trains waiting in the station will not block trains from leaving the station.
```